src/spi.rs

```diff
@@ -165,6 +189,7 @@ pub mod config {
         /// See https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/spi_master.html#timing-considerations
         pub write_only: bool,
         pub duplex: Duplex,
+        pub bit_order: BitOrder,
         pub cs_active_high: bool,
         pub input_delay_ns: i32,
     }
```

Comment:
NIT: for consistency maybe add an pub fn bitorder to the Config builder pattern.

Reply:
Thanks, I missed that 😄 Added it now